apps/dashboard-e2e/src/describer/interpretText/interpretTextDatasets.ts

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IInterpretTextData } from "./IInterpretTextData";
+
+const interpretTextDatasets = {
+  newsgroupBinaryData: {
+    localExplanations: [
```

Comment:
Note: in the future I think we should be able to get this from apps/dashboard directly if we publish it as a package